paper/casingEMpaper.tex

```diff
@@ -31,16 +31,14 @@
 \maketitle
 
 \begin{summary}
-

```

Comment:
@stevejpurves: one more question for you... it doesn't look like the summary is automatically being pulled. Is there something I need to do here? Thanks for your help!

Reply:
glad you got it working! I'm not an expert on the tex->myst conversion, in myst we tags blocks as "abstract" or add it to frontmatter but I don't know what the equivalent is in tex.